src/examples/internals/README.md

```diff
@@ -0,0 +1,5 @@
+# Examples: Internals
+
+This folder contains examples which highlight inner workings and less-documented behaviours of o1js.
+
+These examples might be useful for advanced users and contributors.
```

Comment:
```suggestion
This folder contains examples that highlight the inner workings and less-documented behaviors of o1js.

These examples might be useful for advanced users and contributors.

Run the examples with `npm run build`.
```

Reply:
@mitschabaude It was only yesterday when I saw the description in https://github.com/o1-labs/o1js/issues/1222 that I knew we had runnable examples... we might not need the "Run the examples with..." here, but perhaps at a top-level?

Reply:
that's not correct :D but good idea to add an isntruction how to run examples! I'll add it to the top level examples folder